src/lib/bridge/Bridge.ts

```diff
@@ -117,6 +117,10 @@ export default class Bridge {
     }
 
     private messageHandler(e: MessageEvent): void {
+        if (typeof e === 'undefined' || e.data === 'verify_ready') {
```

Comment:
I think we can do this
```suggestion
        if (typeof !e || e.data === 'verify_ready') {
```

Reply:
I've updated the code by removing `typeof e === 'undefined'` and replaced it with `!e` since `typeof !e` returns `'boolean'` which prevents passing through the if condition.